Create .gitbook.yaml (#313)
mattermost/mattermost-plugin-incident-response: docs/incidents/managing-an-incident.md

# Managing an Incident

## Viewing incident details

> Members are able to do the following via both the right-hand sidebar and the messaging interface:

Author: ```suggestion
Members are able to do the following via both the RHS and the messaging interface:
```

- See who is the incident commander

> - See the remaining steps

Reviewer: Nit: this is completely true, but we can also see the finished steps.

> - See the remaining steps

Author: ```suggestion
- See the remaining steps
- See the finished steps
```

> - Assign a step to self

Reviewer: Again, true, but we can assign the step to anyone else, too.

> - Mark a step as in progress or completed

Reviewer 2: It's no longer possible to model `In progress` -- it's either incomplete, or completed (as seen by the state of the checkbox).

- Start an automated action
- Invite new members to the channel

> ## Assigning steps to members

Author: ```suggestion
```

> Members can assign any step to another member via the right-hand sidebar.

Reviewer: Is this redundant with the item in the list before?

> Members can assign any step to another member via the right-hand sidebar.

Author: ```suggestion
```

## Changing the incident owner

> Any member can assign the incident commander role to themselves in the right-hand sidebar.

Reviewer: Sorry for repeating myself here, but we can assign the commander also to any other person.

> Any member can assign the incident commander role to themselves in the right-hand sidebar.

Author: ```suggestion
An incident member can assign the incident commander role to themselves, or any other incident member, in the RHS.
```
